### DISCUSSION TITLE
安全性/隐私的问题

### DISCUSSION THREAD

**Discussion by xxy0op** [1Panel-dev/1Panel#4541] [General]
![chrome_r799d1cQLn](https://github.com/1Panel-dev/1Panel/assets/109688233/1a2032ba-a194-4e1c-9d5b-185960f28972)
我在登录1panel的时候,需要勾选同意(飞致云社区软件许可协议)中看到的第6条 使用验证 中看到这个 是不是意味着会上传我们网站的数据至外部服务器

**Comment by wanghe-fit2cloud** [comment]
https://github.com/1Panel-dev/1Panel/issues/723#issuecomment-1925776684
